### DISCUSSION TITLE
Can I apply animation to the `tabs` component?

### DISCUSSION THREAD

**Discussion by ojj1123** [chakra-ui/zag#737] [Q&A]
**I wonder if it is possible to apply animation to the `tabs` component as shown below.**

https://github.com/chakra-ui/zag/assets/33178048/20ae77fc-8182-4011-99b1-bc03d90a55c4

[Zag's tabs component](https://zagjs.com/components/react/tabs)
![스크린샷 2023-07-13 02 34 35](https://github.com/chakra-ui/zag/assets/33178048/cdbd0260-74e5-4b8a-ac1d-68a96570f0e0)
![스크린샷 2023-07-13 02 34 53](https://github.com/chakra-ui/zag/assets/33178048/64e8e163-abf0-4197-af43-be5be4759041)

When I checked the HTML elements in the example of the `tabs` component, I found that elements that are not visible in the `tab` have a `hidden` attribute (Red highlight).

The `zag` documentation doesn't show how to apply animation, so I'm posting a question.

1. How can I implement the animation shown in the video above in the `tabs` component? Can I do it?
2. I will use an animation library like [react-spring](https://react-spring.dev/) or [framer-motion](https://www.framer.com/motion/). Is it possible?

**Comment by cschroeter** [comment]
This should be possible using the presence machine. In Ark we have some Animations you can Check Out https://ark-ui.com/docs/react/components/tooltip#adding-animation

**Comment by segunadebayo** [comment]
Closing as outdated — please refer to the updated v1.0 documentation for current API patterns.
